pod/perlexperiment.pod

```diff
@@ -162,6 +152,15 @@ Using this feature triggers warnings in the category C<experimental::try>.
 The ticket for this experiment is
 L<[perl #18760]|https://github.com/Perl/perl5/issues/18760>
 
+=item Use of C<@_> with subroutine signatures
```

Comment:
Shouldn't this be "within signatured subroutines"?

Reply:
Ahyes. Updated, squashed down a bit, and force-pushed. Again.